addon/components/flexberry-layerstree.js

```diff
@@ -0,0 +1,167 @@
+/**
+  @module ember-flexberry-gis
+*/
+
+import Ember from 'ember';
+import layout from '../templates/components/flexberry-layerstree';
+import FlexberryTreeComponent from './flexberry-tree';
+import FlexberryLayersTreenodeComponent from './flexberry-layerstreenode';
+
+/**
+  Component's CSS-classes names.
+  JSON-object containing string constants with CSS-classes names related to component's .hbs markup elements.
+
+  @property {Object} flexberryClassNames
+  @property {String} flexberryClassNames.prefix Component's CSS-class names prefix ('flexberry-layerstree').
+  @property {String} flexberryClassNames.wrapper Component's wrapping <div> CSS-class name ('flexberry-layerstree').
+  @readonly
+  @static
+*/
+const flexberryClassNamesPrefix = 'flexberry-layerstree';
+const flexberryClassNames = {
+  prefix: flexberryClassNamesPrefix,
+  wrapper: flexberryClassNamesPrefix
+};
+
+/**
+  Flexberry layers tree component with [Semantic UI accordion](http://semantic-ui.com/modules/accordion.html) style
+  and ["Data Down Actions UP (DDAU)" pattern](https://emberway.io/ember-js-goodbye-mvc-part-1-21777ecfd708) one way bindings.
+  Component doesn't mutate passed data by its own, it only invokes actions,
+  which signalizes to the route, controller, or another component, that passed data should be mutated.
+  Component must be used in combination with {{#crossLink "FlexberryLayersTreenodeComponent"}}flexberry-layerstreenode component{{/crossLink}},
+  because it's only a wrapper for those tree layers, which are placed on the same tree level.
+
+  Usage:
+  templates/my-form-in-hbs.hbs
+  ```handlebars
+  {{#flexberry-layerstree}}
+    {{#flexberry-layerstreenode
+      name="hbsLayer1 (it is not a leaf)"
+      type="wms"
+      visibility=true

```

Comment:
It would be wise to add actions to explicitly defined "flexberry-layerstreenode" components: (visibilityChange=(action  "onHbsLayer1VisibilityChange" "hbsLayer1.visibility") headerClick=(action "OnHbsLayer1HeaderClick") "hbsLayer1"),
to show user the difference between hbs-defined layers tree nodes & JOSN-defined layers tree nodes.